Rego/SharepointConfig.rego

```diff
@@ -240,14 +328,13 @@ tests[{
     Policy := input.SPO_tenant[_]
     [ErrMsg, Status] := ExpirationTimersVerificationCode(Policy)
 }
-#--
 
 ###################
-# MS.SHAREPOINT.5 #
+# MS.SHAREPOINT.4 #
 ###################
 
 #
-# MS.SHAREPOINT.5.1v1
+# MS.SHAREPOINT.4.1v1
 #--
```

Comment:
Should be a newline after this line to break up declaration of start of new policy & comment notes for code